README.md

```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
 
+## Requirements
+
+### Usage Requirements
+
+Please see the [Setup Process](#setup-process) section
+
+### Development Requirements
+
+This library needs these to be in order to build and run:
+
+- Node 20
+- Typescript 4+
+- Playwright
+- [Wrangler](https://developers.cloudflare.com/workers/wrangler/install-and-update/) v3+
+
 ## Setup process
```

Comment:
* This seems outdated, you don't need to contact support to setup the Cloudflare integration. 
* The Readme should point people to https://dev.fingerprint.com/docs/cloudflare-integration follow the set up guide using the installation wizard.

Reply:
Updated here